### PR TITLE
integrated Jorgensen data

### LINKED ISSUE #32: Add Jorgensen WNAI data
These `dplace check` errors should go away:

* ERROR   sicoli_and_holton2014: invalid soc_id J34
* ERROR   sicoli_and_holton2014: invalid soc_id J4
* ERROR   sicoli_and_holton2014: invalid soc_id J38
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J85
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J86
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J87
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J135
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J83
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J81
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J159
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J79
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J82
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J117
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J72
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J71
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J114
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J115
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J116
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J118
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J146
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J145
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J84
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J124
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J133
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J134
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J132
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J131
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J129
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J130
* ERROR   dunn_et_al2011_utoaztecan: invalid soc_id J78
* ERROR   sicoli_and_holton2014: invalid soc_id J152
* ERROR   sicoli_and_holton2014: invalid soc_id J153
* ERROR   sicoli_and_holton2014: invalid soc_id J156
* ERROR   sicoli_and_holton2014: invalid soc_id J155
* ERROR   sicoli_and_holton2014: invalid soc_id J147
* ERROR   sicoli_and_holton2014: invalid soc_id J148
* ERROR   sicoli_and_holton2014: invalid soc_id J149
* ERROR   sicoli_and_holton2014: invalid soc_id J150
* ERROR   sicoli_and_holton2014: invalid soc_id J151
* ERROR   sicoli_and_holton2014: invalid soc_id J93
* ERROR   sicoli_and_holton2014: invalid soc_id J94
* ERROR   sicoli_and_holton2014: invalid soc_id J35
* ERROR   sicoli_and_holton2014: invalid soc_id J40
* ERROR   sicoli_and_holton2014: invalid soc_id J42
* ERROR   sicoli_and_holton2014: invalid soc_id J43
* ERROR   sicoli_and_holton2014: invalid soc_id J41
* ERROR   sicoli_and_holton2014: invalid soc_id J157
* ERROR   sicoli_and_holton2014: invalid soc_id J158
* ERROR   sicoli_and_holton2014: invalid soc_id J2
* ERROR   sicoli_and_holton2014: invalid soc_id J1

### PR DESCRIPTION
**Pull request by xrotwang**
This PR integrates the Jorgensen data sent by @kirbykat in September 2017.

Closes #32

### CODE REVIEW

**Comment by xrotwang** [comment]
Note that locations for the WNAI societies are still lacking.

**Comment by xrotwang** [comment]
@Bibiko import into the dplace app works already with this data, so feel free to test.

File: phylogenies/bowern_and_atkinson2012/taxa.csv [hunk 1]
```diff
@@ -146,7 +146,7 @@ Ritharrngu,,,
 SouthernWalmajarri,,,
 Thalanyji,,,
 Tharrgari,,,
-Umpila,ump1239,xd945,B93
+Umpila,umpi1239,xd945,B93
```
Comment (xrotwang) [inline phylogenies/bowern_and_atkinson2012/taxa.csv]: This seems to have been a typo.

**Comment by SimonGreenhill** [comment]
I'll add the locations as a new issue.

**Pull request merged**